src/js/controllers/navbar.js

```diff
@@ -60,12 +60,16 @@ module.controller('NavbarCtrl', ['$scope', '$element', '$compile', 'rpId',
   });
 
   var updateNotifications = function() {
-    if ($scope.events) {
+    if (!$scope.connected) {
```

Comment:
if the client suddenly goes offline and then comes back online we don't want the notifications to disappear and then appear again. Let's do the check with `$scope.onlineMode`